ui/src/main/webapp/src/app/components/navigation/breadcrumbs.component.html

```diff
@@ -1,6 +1,42 @@
+
+<style>
+ol.breadcrumb {
+    /* Pull the line to the borders. */
+    margin-left:  -30px;
+    margin-right:  -30px;
+    padding-left: 30px;
+    padding-right: 30px;
+    border-bottom: 1px solid #d5d5d5;
+    margin-bottom: 0px;
+    background-color: white;
+}
+
+ol.breadcrumb > li {
+    font-size: 10pt;
+}
+
+ol.breadcrumb > li:last-child > a,
+ol.breadcrumb > li:last-child > a:hover {
+    font-weight: bold;
+    text-decoration: none;
+    color: dimgrey;
+    cursor: default !important;
+}
+</style>
```

Comment:
I think this is a better way. Keeping visual style with the template is desired. Also, the result is the same - angular adds it to the DOM as a <style> block.